ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml

```diff
@@ -183,6 +183,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-sdn
+- as: e2e-aws-ipi-disconnected-private-p2
```

Comment:
Should we add some "arm" keyword here?

Reply:
yes, a good point. @aleskandro  is it ok that I change all the profiles name with arm keyword in file openshift-openshift-tests-private-release-4.12__arm64-nightly.yaml in this PR

Reply:
@jianlinliu Checked with Alessandro, the run will add arm64 keywords in job name automatically and arm64 profiles are in a separate file, it's ok to keep it as it is

Reply:
Since the file name is *_arm64-*, so the job name is "periodic-ci-openshift-openshift-tests-private-release-4.12-arm64-nightly-e2e-aws-ipi-disconnected-private-p2", which combines specific prefix `periodic-ci` + file name + job name. It already indicates it's for arm